### PR TITLE
Fix layout issues

### PR DESCRIPTION
**Pull request by thiagomurakami**
### What problem is this solving?

Fixes some layout inconsistencies:
#### 1. Overflow in horizontal axis:

**Before:**
Screen Shot 2020-04-28 at 19 41 57

**After:**
Screen Shot 2020-04-28 at 19 42 01

#### 2. A one pixel line right beside the summary:

**Before:**
Screen Shot 2020-04-28 at 19 57 16

**After:**
Screen Shot 2020-04-28 at 19 57 19

#### 3. Drawer overflow.
**Before:**
Screen Shot 2020-04-17 at 13 40 32

**After:**
Screen Shot 2020-04-17 at 13 40 44

Also added a simple animation to the summary:
ezgif com-video-to-gif

#### How should this be manually tested?

[Workspace](https://fixlayout--checkoutio.myvtex.com/checkout#/)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes

[ch35949]

### CODE REVIEW

**Comment by marcoskwkm** [comment]
Nice!! I have some comments which might not necessarily be related to this PR, but since you're editing the layout I thought it would be worth mentioning.

i. In one of the screen sizes the two "Ver Carrinho" buttons are not aligned.
<p align="center"></p>

ii. If I click the blue "Ver Carrinho" button near the summary, the drawer does not show the totalizers. It does when I click the gray one on top.
<p align="center"></p>

**Comment by lucasecdb** [comment]
@marcoskwkm the second point is actually by design. since the button to open the drawer is already inside the summary, the summary should not be in the drawer content

**Comment by lucasecdb** [comment]
@marcoskwkm about the first point, I looked at figma to see if that was intentional or just a mistake on my part, but apparently it's supposed to be like that. in figma the "View cart" on the top is outdated (it used to be an icon with the number of items in the cart next to it), but I think its intented to be aligned like this:

image

**Review by marcoskwkm — APPROVED** [review]
@lucasecdb Gotcha. Thanks for the clarifications!

I'm not very knowledgeable in CSS but LGTM.

File: styles/css/vtex.store-drawer.css [hunk 1]
```diff
@@ -5,6 +5,11 @@
 
 .drawer--checkout {
   padding: 1.5rem 1rem;
+  overflow-y: auto;
+}
+
+.drawerContent--checkout {
+  overflow-y: visible !important;
```
Comment (augustocb) [inline styles/css/vtex.store-drawer.css]: Does it need to have `!important` property? 😕

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Aded
```
Comment (lucasecdb) [inline CHANGELOG.md]:
```suggestion
### Added
```

File: styles/css/vtex.flex-layout.css [hunk 3]
```diff
@@ -43,8 +28,8 @@
 
 .flexCol--itemsSummary,
 .flexCol--itemsSummaryMobile {
-  background-color: #f2f4f5;
   height: 100%;
+  animation: fade-in-right 500ms cubic-bezier(0.455, 0.030, 0.515, 0.955) 480ms both;
```
Comment (lucasecdb) [inline styles/css/vtex.flex-layout.css]: this animation is also happening when the drawer is opened, which I don't think is intended right?

File: styles/css/vtex.store-drawer.css (hunk 1 above)
Comment (thiagomurakami) [reply styles/css/vtex.store-drawer.css]: Yes, there's an inline `overflow-y: scroll;`:
Screen Shot 2020-04-29 at 17 11 51

File: styles/css/vtex.flex-layout.css (hunk 3 above)
Comment (thiagomurakami) [reply styles/css/vtex.flex-layout.css]: Yea! Nice catch

**Review by lucasecdb — APPROVED** [review]
LGTM! :sparkles:

File: styles/css/vtex.store-drawer.css (hunk 1 above)
Comment (augustocb) [reply styles/css/vtex.store-drawer.css]: what the hack

**Comment by augustocb** [comment]
@thiagomurakami I found an issue in summary's layout on a large screen:

Screen Shot 2020-04-30 at 11 21 45

File: styles/css/vtex.store-drawer.css (hunk 1 above)
Comment (thiagomurakami) [reply styles/css/vtex.store-drawer.css]: https://github.com/vtex-apps/drawer/blob/master/react/Drawer.tsx#L199 :(

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**